scripts/initdata.py

```diff
@@ -63,26 +62,26 @@ def ensure_user(user):
         'description': 'description IT news',
         'name': 'IT news',
         'is_active': 1,
-        'geo_location': 11,
+        'geo_location': '0',
     },
     {
         'description': 'description Comp@Tech',
         'name': 'Comp@Tech',
         'is_active': 1,
-        'geo_location': 1,
+        'geo_location': '0',
     },
     {
         'description': 'description Game news',
         'name': 'Game news',
         'is_active': 0,
-        'geo_location': 1,
+        'geo_location': '0',
     },
 ]
 
 questions = [
     {
         'text': 'Did you like the lecture?',
-        'type': 'yesno',
+        'status': 'yesno',
```

Comment:
why status? type looks more meaningful here

Reply:
This is status qustion. It is not type question